Makefile

```diff
@@ -24,7 +24,7 @@ GOTEST    := GO111MODULE=on CGO_ENABLED=1 $(GO) test -p 3
 ARCH      := "`uname -s`"
 LINUX     := "Linux"
 MAC       := "Darwin"
-PACKAGES  := $$(go list ./...| grep -vE 'vendor|cmd|test|proto|diff|bin')
+PACKAGES  := $$(go list ./...| grep -vE 'vendor|cmd|test|proto|diff|bin|fuzz')
```

Comment:
where's the fuzz pkg?

Reply:
@july2993 Not included here. I'm thinking how to introduce fuzz test, whether directly here or as a different repository, since it has amassed over 3000 corpus already.